app/models/log.rb

```diff
@@ -106,7 +106,12 @@ def blank_invalid_non_setup_fields!
       errors.each do |error|
         next if setup_ids.include?(error.attribute.to_s)
 
-        public_send("#{error.attribute}=", nil)
+        question = form.questions.find { |q| q.id == error.attribute.to_s }
+        if question.type == "checkbox"
+          question.answer_keys_without_dividers { |attribute| public_send("#{attribute}=", nil) }
```

Comment:
Are you missing an enumerator here? If so, please could you add a spec for this change?

```suggestion
          question.answer_keys_without_dividers.each { |attribute| public_send("#{attribute}=", nil) }
```

Reply:
Are these changes specific to bulk upload, or should they be applicable for single log submission as well?